ui/src/shared/components/ViewSwitcher.tsx

```diff
@@ -69,7 +69,7 @@ const ViewSwitcher: FunctionComponent<Props> = ({
           )}
         </LatestValueTransform>
       )
-
+    case 'check':
```

Comment:
There exists a `CheckPlot` component that I think should be used here instead.

Reply:
Agreed! https://app.zenhub.com/workspaces/applicationmonitoring-5d2e06177e9542541ca076ad/issues/influxdata/influxdb/14537